templates/zerver/invalid_realm.html

```diff
@@ -0,0 +1,9 @@
+{% extends "zerver/portico.html" %}
+{% block portico_content %}
+
+<h3>{{ _('Realm Does Not Exist') }}</h3>
+
+<p>{{ _('Hi there! Thank you for your interest in Zulip') }}.</p>
```

Comment:
Let's add a period at the end: `Hi there! Thank you for your interest in Zulip.`